modules/instance/outputs.tf

```diff
@@ -0,0 +1,7 @@
+#########################################################################################
+# IBM Cloud Key Management Services Provisioning and Managing Keys
+# Copyright 2021 IBM
+#########################################################################################
+output "kms_instance_output" {
```

Comment:
What will be the out of this module (sorry I am not sure)

One of the guidelines is to capture the id and guid of each module as output